crates/vhost-device-gpio/src/mock_gpio.rs

```diff
@@ -51,11 +51,11 @@ impl MockGpioDevice {
 }
 
 impl GpioDevice for MockGpioDevice {
-    fn open(_device: u32) -> Result<Self>
+    fn open(ngpios: u32) -> Result<Self>
```

Comment:
The `open` contract on `GpioDevice` is really weird... Here it forces us to take a u32 only to convert it to u16 internally... Of course fine within this PR :) Maybe I will send a PR after this to clean it up...